db/migrate/20231210061330_add_created_at_to_fts_targets_with_index.rb

```diff
@@ -0,0 +1,105 @@
+# For auto load
+FullTextSearch::Migration
+
+class AddCreatedAtToFtsTargetsWithIndex < ActiveRecord::Migration[5.2]
+  def up
```

Comment:
Could you skip all `up`/`down` processes when `!table_exists?(:fts_targets)`?
If Mroonga table is broken (`!table_exist?(:fts_targets)`), we can't run migration successfully. But we want to run migration to recreate DB: https://github.com/clear-code/redmine_full_text_search#mroonga ("Recreate schema for this plugin:")
If we don't have the check, `RAILS_ENV=production bin/rails redmine:plugins:migrate NAME=full_text_search VERSION=0` will be failed in the situation.

Reply:
fix: [592fb41](https://github.com/clear-code/redmine_full_text_search/pull/118/commits/592fb41179cd74d47013c6a049659abe9cccdc10)